test_env/test_key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
This is late, but couldn't you use the Dockerfile to generate these keys? Then copy the private one out to the host?

```
RUN ssh-keygen <arguments>
```

```
docker cp test-env:/path/of/private/key /test_env
```

Just a thought, would keep them out of the repo.

Reply:
Hmmmm perhaps, but I think its a bit cleaner to always use the same key